docker/configuration.py

```diff
@@ -0,0 +1,22 @@
+# This is a list of valid fully-qualified domain names (FQDNs) for this server.
+# The server will not permit write access to the server via any other
+# hostnames. The first FQDN in the list will be treated as the preferred name.
+#
+# Example: ALLOWED_HOSTS = ['peering.example.com', 'peering.internal.local']
+import os
+ALLOWED_HOSTS = ["*"]
+
+# Must be unique to each setup (CHANGE IT!).
+SECRET_KEY = "ef7npku*djrj_r4jt4cojo8^j@2($$@05e(eq_mn!ywx*jg0vy"
```

Comment:
Should probably have this passed in via an ENV var

Reply:
I haven't seen any traction on this PR. If you'd like I can take over and make the edit to get the secret from an ENV var.